Tests/ToucanSDKTests/Utilities/UnboxingTestSuite.swift

```diff
@@ -4,191 +4,193 @@
 //
 //  Created by Viasz-Kádi Ferenc on 2025. 05. 09..
 //
-
-import Foundation
-import Testing
-@testable import ToucanModels
-@testable import ToucanContent
-
-@Suite
-struct UnboxingTests {
-
-    @Test
-    func unboxing() throws {
```

Comment:
Tests not yet restoredreplaced with E2E tests. I think we don't test this anywhere else.